public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -274,7 +287,7 @@ const update = (data) => {
     //Add the X Axis for the total failed sms graph
     total_failed_sms_graph.append("g")
         .attr("transform", "translate(0," + Height + ")")
-        .call(d3.axisBottom(x)
+        .call(d3.axisBottom(x_axis_failed_sms_range)
```

Comment:
Sorry to go round on this again, but on second reading today I think `failed_messages_x_axis_range` would read even better

Reply:
no worries :)

Reply:
I will rename for attributes for y axis in another branch